content/en/tracing/troubleshooting/tracer_debug_logs.md

```diff
@@ -220,7 +220,7 @@ If there are errors that you don't understand, or if traces are reported as flus
 {{< /programming-lang >}}
 {{< programming-lang lang="python" >}}
 
-For more visibility, include `DD_LOGGING_RATE_LIMIT=0`.
+For more visibility, include `DD_TRACE_LOGGING_RATE=0`.
```

Comment:
Is this needed anymore? at least when debug logging is enabled we disable the rate limit (iirc)?

Reply:
Hmm I don't see any logic for this in the library. Only place that the rate limit appears to be set is through the environment variable.